keps/sig-cloud-provider/0004-cloud-provider-template.md

```diff
@@ -0,0 +1,80 @@
+---
+kep-number: 4
+title: Cloud Provider <my cloud provider>
+authors:
+  - "@janedoe"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-aaa
+  - sig-bbb
+reviewers:
+  - TBD
+  - "@alicedoe"
+approvers:
+  - "@andrewsykim"
+  - "@hogepodge"
+  - "@jagosan"
+editor: TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional
+see-also:
+  - KEP-1
+  - KEP-2
+replaces:
+  - KEP-3
+superseded-by:
+  - KEP-100
+---
+
+# Cloud Provider <my cloud provider>
+
+This is a KEP template, outlining how to propose a new cloud provider into the Kubernetes ecosystem.
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Requirements](#requirements)
+* [Proposal](#proposal)
+
+## Summary
+
+This is where you add a summary of your cloud provider and other additional information about your cloud provider that others may find useful.
+
+### Goals [optional]
```

Comment:
I think this was intended to be nested under `Motivation` which I like. I think the motivation should also be listed as required